stats/PA1673_gradient/tybalt_2layer_10latent_stats.csv

```diff
@@ -0,0 +1,201 @@
+	loss	val_loss	learning_rate	batch_size	epochs	kappa
```

Comment:
This should be a .tsv not .csv. Also seems like the index column should not be saved here